projects/storefrontlib/cms-components/myaccount/address-book/address-book.component.ts

```diff
@@ -30,10 +32,14 @@ export class AddressBookComponent implements OnInit {
   showEditAddressForm = false;
   editCard: string | null;
 
+  /**
+   * TODO: (#CXSPA-53) Remove featureConfigService from constructor in 6.0
+   */
   constructor(
-    public service: AddressBookComponentService,
+    protected service: AddressBookComponentService,
```

Comment:
Not sure why this has changed from `public` to `protected` but worth mentioning in migration docs.

Reply:
change back to public